src/components/screens/Feed/components/FeedView.tsx

```diff
@@ -87,6 +93,14 @@ function FeedView({ feed, community = false, header }: FeedViewProps) {
 
         return (
           <HStack space={1}>
```

Comment:
Only thing besides what Sean asked is to probably made the space here 2 instead of 1. Seems people have trouble sometimes actually pressing the icon. To be fair, they are pretty close together. Not your fault, just something that needs to get changed and since you're editing this anyway we can knock it out here.

Thanks!